### ISSUE TITLE
Tests are silently failing

### ISSUE THREAD

**Issue by edoroshenko** [postcss/postcss-nested#53]
Looks like something is broken. I've just cloned the repo, installed everything and run tests. The report is successful, but there's some errors in console like
```
(node:95861) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): Error: expect(received).toEqual(expected)

Expected value to equal:
  "a b .one {}
a b .two {}"
Received:
  "a b .one {} a b .two {}"
(node:95861) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 2): Error: expect(received).toEqual(expected)

Expected value to equal:
  "@media screen {
    a b {
        width: auto
    }
}"
Received:
  "@media screen {a b { width: auto } }"
(node:95861) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 3): Error: expect(received).toEqual(expected)

Expected value to equal:
  "@media screen {
    a b {
        color: black
    }
}"
```
So I went there after I got errors in my project. Rolled back to 1.0.1, it works

**Comment by ai** [comment]
Oops, I forgot `return`. Wait a second, I will fix it. Thanks for report.

**Comment by ai** [comment]
Fixed 042847d

**Comment by ai** [comment]
Released in 2.0.2
